### DISCUSSION TITLE
Neuron development guide

### DISCUSSION THREAD

**Discussion by Keith-CY** [nervosnetwork/neuron#2394] [Announcements]
There are 3 guidelines to make Neuron better and more aggressive.

1. Follow up RFC actively and push other applications in CKB Ecosystem to catch up with Neuron;
2. Support DApp signing;
3. Integrate with light client protocol to make Neuron lighter.

**Comment by Keith-CY** [comment]
Also `Neuron Lite` was rejected because it would be not decentralized enough if Neuron/Neuron Lite are maintained by the same team

@yanguoyu @qiweiii

**Comment by yanguoyu** [comment]
> 
How does the light client protocol work?

**Reply by Keith-CY** [reply]
It's quite similar to synchronization of a full node but only block headers will be downloaded and used to verify users' transaction status.

Because block header can be regarded as a digest of its block, transactions in the block are also packaged into the header.

So if there's a block header indicating that your transaction has been committed, it's committed in that block and probably accepted by the chain(we use `probably` here because POW is of probabilistic certainty).

More details can be found in `Simplified Payment Verification` in the bitcoin whitepaper.
